Components/default.css

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
 * {
   margin: 0;
   padding: 0;

```

Comment:
other then that, to me everything looks good, 
just a suggestion- you can write comments above links inserted.